public/apps/configuration/utils/list-state-utils.tsx

```diff
@@ -0,0 +1,123 @@
+/*
```

Comment:
Thanks for extracting the utils functions. Looks like they are very generic and could potentially be reused by other pages. Just want to confirm that "list-state" means "array"?

Reply:
Yep, good point. I should follow the javascript naming convention using `array` instead of `list`.